swift/ql/lib/codeql/swift/dataflow/Ssa.qll

```diff
@@ -169,7 +169,8 @@ module Ssa {
       exists(VarDecl var, SsaInput::BasicBlock bb, int blockIndex, ConditionElement ce, Expr init |
         this.definesAt(var, bb, blockIndex) and
         ce.getPattern() = bb.getNode(blockIndex).getNode().asAstNode() and
-        init = ce.getInitializer()
+        init = ce.getInitializer() and
+        strictcount(Ssa::WriteDefinition alt | alt.definesAt(_, bb, blockIndex)) = 1 // exclude cases where there are multiple writes from the same pattern, this is at best taint flow.
```

Comment:
I imagine we could also write this by inspecting the structure of `ce.getPattern()`, but I appreciate the simplicity of this solution 👍.